### DISCUSSION TITLE
Standardize table time fields

### DISCUSSION THREAD

**Discussion by downdawn** [fastapi-practices/fastapi_best_architecture#136] [Ideas]
1、Each table, except for intermediate tables, should have create_time and update_time fields.

2、create_user and update_user fields are optional.

3、User table:
time_joined -> join_time
last_login -> last_login_time
user_uuid -> uuid
create_time
update_time

4、func.now() defaults to UTC time, but it should actually be the server time or Shanghai timezone.

**Comment by wu-clan** [comment]
Positive content, I think we should make some fixed trade-offs

`create_user` and `update_user` 
1. in BaseMixin, but all are optional fields
2. Make them as separate Mixin, create_user mandatory, update_user optional

`create_time` and `update_time`
Provided as BaseMixin, required

`user table`
Agree to change them, those naming are unsurprisingly django's default

`datetime`
A separate time tool should be created with support for time zone settings, and global use of time zone time is a headache

**Reply by downdawn** [reply]
Yes. The second option might be better.

**Comment by wu-clan** [comment]
#152
